src/copy.c

```diff
@@ -45,15 +46,78 @@
 #include "nodes/chunk_insert_state.h"
 #include "subspace_store.h"
 
+/* PG14_GE is defined in compat/compat.h, so it must be included first */
```

Comment:
that comment is kinda redundant since this is true for every macro and we dont put similar comments for those

Reply:
The comment is now removed.